twtr/grails-app/assets/javascripts/app/controllers/userDetail-controller.js

```diff
@@ -68,4 +115,7 @@ angular.module('app').controller('userDetailController', function($scope, $locat
                 $scope.getAccount();
             })
     };
+
+    if($scope.auth.account == null)
```

Comment:
why is this needed since the auth.account was already checked and used previously